src/dataflow/src/render/sinks.rs

```diff
@@ -113,20 +113,26 @@ where
 
         let keyed = if user_key_indices.is_some() {
             let key_indices = user_key_indices.expect("known to exist");
+            let mut datum_vec = repr::DatumVec::new();
             collection.map(move |row| {
                 // TODO[perf] (btv) - is there a way to avoid unpacking and repacking every row and cloning the datums?
                 // Does it matter?
```

Comment:
is this comment (and the next one in this file) now obsolete/need updating?

Reply:
I think it is still as valid. The "unpacking" still happens, but it doesn't allocate any more.